fontc/src/change_detector.rs

```diff
@@ -72,7 +72,7 @@ impl ChangeDetector {
             .glyph_name_filter
             .map(|raw_filter| Regex::new(&raw_filter))
             .transpose()
-            .map_err(Error::BadRegex)?;
+            .expect("validated during arg parsing");
```

Comment:
If we ever launch other than by CLI this can still happen so I suggest leaving the error despite it being much less likely now

Reply:
do we ever or might we ever do that, outside of a test?

Reply:
We might in future